src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1000,11 +1000,17 @@ public void GetAsync_ServerNeedsAuthAndNoCredential_StatusCodeUnauthorized()
             }
         }
 
-        [ActiveIssue(22187, TargetFrameworkMonikers.Uap)]
         [OuterLoop] // TODO: Issue #11345
```

Comment:
Since we are changing this code right now, would it make sense to remove the TODO? it seems that the issue is closed:
https://github.com/dotnet/corefx/issues/11345

Reply:
That particular TODO we still have everywhere in the code. The issue itself was closed.  But it is a larger issue across the entire codebase.

We really just need to change that particular TODO into a string reason about why the tests are outerloop.  But we'll do that in a separate future PR.

cc: @stephentoub